state-machine.md

```diff
@@ -55,30 +58,50 @@ func ApplyMessage(st StateTree, msg Message) MessageReceipt {
 		Fatal("invalid nonce")
 	}
 
+  toActor, found := st.GetActor(msg.To)
+  if !found {
+    toActor = TryCreateAccountActor(st, msg.To)
+  }
+  
 	st.DeductFunds(msg.From, totalCost)
 	st.DepositFunds(msg.To, msg.Value)
 
 	vmctx := makeVMContext(st, msg)
 
-	ret, errcode := fromActor.Invoke(vmctx, msg.Method, msg.Params)
-	if errcode != 0 {
-		// revert all state changes since snapshot
-		st.Revert()
-		st.DeductFunds(msg.From, vmctx.GasUsed() * msg.GasPrice)
-	} else {
-		// refund unused gas
-		st.DepositFunds(msg.From, (msg.GasLimit - vmctx.GasUsed()) * msg.GasPrice)
-	}
+  if msg.Method != "" {
+		ret, errcode := toActor.Invoke(vmctx, msg.Method, msg.Params)
+		if errcode != 0 {
+			// revert all state changes since snapshot
+			st.Revert()
+			st.DeductFunds(msg.From, vmctx.GasUsed() * msg.GasPrice)
+		} else {
+			// refund unused gas
+			st.DepositFunds(msg.From, (msg.GasLimit - vmctx.GasUsed()) * msg.GasPrice)
+		}
+  }
 
 	// reward miner gas fees
-	st.DepositFunds(msg.To, msg.GasPrice * vmctx.GasUsed())
+	st.DepositFunds(BlockMiner, msg.GasPrice * vmctx.GasUsed())
 
 	return MessageReceipt{
 		ExitCode: errcode,
 		Return: ret,
 		GasUsed: vmctx.GasUsed(),
 	}
 }
+
+func TryCreateAccountActor(st StateTree, addr Address) Actor {
```

Comment:
For clarity's sake, it might be best to leave this definition out of this spec (in the way the methods on the state tre are undefined). Maybe it it's worth mentioning in the address section.